ComponentKit/Core/Scope/CKComponentScope.h

```diff
@@ -65,6 +65,13 @@ class CKComponentScope {
   */
   CKComponentStateUpdater stateUpdater(void) const;
 
+  /**
+   @return The scope handle associated with this scope.
+   @discussion This is exposed for use by the framework. You should almost certainly never call this for any reason
+               in your components.
+   */
+  CKComponentScopeHandle *scopeHandle(void) const;
+
```

Comment:
How? C++ doesn't support this I think without a friend type, and I don't like the indirection.